### PR TITLE
New package: reddio-0.3

### LINKED ISSUE #14223: Package request: reddio
- package: https://gitlab.com/aaronNG/reddio

a command-line interface for Reddit written in POSIX sh
```
# Dependencies: curl for downloading from the reddit api 
                coreutils, sed 
# jq for parsing json data 

# Optional: nc for authorizing this client
```

### PR DESCRIPTION
**Pull request by Itamiko**
resolves #14223

### CODE REVIEW

File: srcpkgs/reddio/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'reddio'
+pkgname=reddio
+version=0.3
+revision=5
+wrksrc="reddio-v0.3"
```
Comment (jnbr) [inline srcpkgs/reddio/template]: use `wrksrc="reddio#v${version}"`, so you don't have to change it on every update

File: srcpkgs/reddio/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'reddio'
+pkgname=reddio
+version=0.3
+revision=5
```
Comment (jnbr) [inline srcpkgs/reddio/template]: should be 1 for a new package

File: srcpkgs/reddio/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+wrksrc="reddio-v0.3"
+depends="curl jq"
+short_desc="Command-line interface for Reddit written in POSIX"
+maintainer="itamiko <itamiko@pm.me>"
+license="MIT"
+homepage="https://gitlab.com/aaronNG/reddio"
+distfiles="https://gitlab.com/aaronNG/reddio/-/archive/v${version}/reddio-v${version}.tar.gz"
+checksum=be200dfba2bbbbff8ba2445ad9f41842419c9e3d65aa103a8334886f2b1ff870
+patch_args="-Np1"
```
Comment (jnbr) [inline srcpkgs/reddio/template]: you could use the default p0 if you remove the leading `a/` and `b/` from the paths in the patch

File: srcpkgs/reddio/template [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+maintainer="itamiko <itamiko@pm.me>"
+license="MIT"
+homepage="https://gitlab.com/aaronNG/reddio"
+distfiles="https://gitlab.com/aaronNG/reddio/-/archive/v${version}/reddio-v${version}.tar.gz"
+checksum=be200dfba2bbbbff8ba2445ad9f41842419c9e3d65aa103a8334886f2b1ff870
+
+do_install() {
+	vbin reddio
+	vmkdir usr/share/reddio
+	vcopy share/reddio/** usr/share/reddio/
```
[the remark this answers is not in the document]
Comment (Itamiko) [reply srcpkgs/reddio/template]: no need to quote them. `./xbps-src show-files reddio` shows all files in their respective places

File: srcpkgs/reddio/reddio.INSTALL.msg [hunk 5]
```diff
@@ -0,0 +1 @@
+Optional: nc to authenticate with your Reddit account credentials
```
Comment (Chocimier) [inline srcpkgs/reddio/reddio.INSTALL.msg]: Please add some names of packages that contain nc.

File: srcpkgs/reddio/template (hunk 4 above)
[the remark this answers is not in the document]
Comment (Chocimier) [reply srcpkgs/reddio/template]: vcopy copies recursively, do not expand recursively with shell.

File: srcpkgs/reddio/template [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+post_install() {
+	 vlicense LICENSE
```
Comment (Chocimier) [inline srcpkgs/reddio/template]: move into do_install

File: srcpkgs/reddio/reddio.INSTALL.msg (hunk 5 above)
Comment (jnbr) [reply srcpkgs/reddio/reddio.INSTALL.msg]: Or just drop the INSTALL.msg. It seems that most people don't read them anyway.
I think we should use them for really important stuff only, so there is a higher signal/noise ratio and a better chance of people reading them.

**Comment by ndowens** [comment]
Is this finished/being worked on?

**Comment by Itamiko** [comment]
no, I will rewrite the build template tonight or tomorrow since right now its 22:00 atm
 @ndowens

**Comment by Itamiko** [comment]
closing as source pull repository has been deleted. Will resubmit with pull request soon

**Comment by Itamiko** [comment]
closing as source pull repository has been deleted. Will resubmit with pull request soon

**Pull request closed without merging**